libraries/EmbeddedChessPieces/src/ECPMovement/ECPMovement.h

```diff
@@ -0,0 +1,74 @@
+/**
+ * @file ECPMovement.h
+ * @author Ines Rohrbach, Nico Schramm
+ * @brief Helper class for chess piece movement.
+ * @version 0.1
+ * @date 2024-11-12
+ * 
+ * @copyright Copyright (c) 2024
+ * 
+ */
+
+#ifndef ECPMovement_h
+#define ECPMovement_h
+
+#include <Dezibot.h>
+#include <EmbeddedChessPieces.h>
+
+#define FORWARD_TIME 750
+#define MOVEMENT_BREAK 375
+
+#define CALIBRATION_STRAIGHT 4050

```

Comment:
We might want to put this in the constructor as well with a default value. When using it later on, it might be uncomfortable to have to change it in the header file, especially when flashing multiple bots.